fava/static/css/journal-table.css

```diff
@@ -136,6 +137,7 @@
 
 .journal .datecell {
   width: 5.5rem;
+  white-space: nowrap;
```

Comment:
Shouldn't the date cell always have enough space?

Reply:
In Chrome and Safari sometimes it doesn't.

Reply:
Does it look bad with the nowrap setting then? We could also make this cell minimally wider

Reply:
:+1: